docs/dev/checks/README.md

```diff
@@ -8,6 +8,12 @@ fit for your use case, please [refer to the official documentation][custom-check
 Every check has its own YAML configuration file. The file has one mandatory key,
 `instances` and one optional, `init_config`.
 
+Note: If you want to run the custom check inside your development workspace 
```

Comment:
I feel like that the corollary to this is that the proper location is `bin/agent/dist`? Wondering if this might mislead people into thinking dropping the checks and config files in `dev/dist` is enough if they read this quickkly. I would maybe simplify/reorder this message to say:
```
Note:
If you want to run a custom check inside your development workspace (github.com/DataDog/datadog-agent), you must put `MyCheck.yaml` and `MyCheck.py` in the `bin/agent/dist` folder located at the root of the datadog-agent repository. 
Please keep in mind that the `invoke agent.build` task will copy the contents inside `dev/dist` to `bin/agent/dist` when ran, so you can use that path if you need a longer-lived location for your custom checks. 
```
Or something along those lines?